src/modules/feature-modules/assessment/pages/innovation/overview/overview.component.ts

```diff
@@ -69,7 +69,7 @@ export class InnovationOverviewComponent extends CoreComponent implements OnInit
         total: statistics[InnovationStatisticsEnum.SECTIONS_SUBMITTED_SINCE_ASSESSMENT_START_COUNTER].total,
         footer: `Last submitted section: "${this.translate('shared.catalog.innovation.innovation_sections.' + statistics[InnovationStatisticsEnum.SECTIONS_SUBMITTED_SINCE_ASSESSMENT_START_COUNTER].lastSubmittedSection)}"`,
         date: statistics[InnovationStatisticsEnum.SECTIONS_SUBMITTED_SINCE_ASSESSMENT_START_COUNTER].lastSubmittedAt,
-        emptyMessage: `You haven't had submitted sections since the assessment started.`
+        emptyMessage: `No sections habe been submitted since the supported started.`
```

Comment:
@Emanuel-Mariano-BJSS typo here 😄

Reply:
I don't think it makes sense to tell the QA he hasn't submitted sections though, since it's the innovator who submits sections 🤔